tasks/create-github-release/create-github-release.yaml

```
@@ -40,9 +40,7 @@ spec:
         
         cd "$(workspaces.data.path)/$CONTENT_DIRECTORY"
         set -o pipefail
-        gh release create v$RELEASE_VERSION *.zip *.json *SHA256SUMS --repo $REPOSITORY | tee $(results.url.path)
-        # We will add .sig file when we have the sigining step is done
-        #gh release create v$RELEASE_VERSION *.zip *.json *.sig --repo $REPOSITORY 
+        gh release create v$RELEASE_VERSION *.zip *.json *SHA256SUMS *.sig --repo $REPOSITORY | tee $(results.url.path)
```

I wonder if there shouldn't be some checks for the files we're trying to copy. Because if, for example, there is no sig file there: `gh` will still be called, but with literally `*.sig` and then it will fail on a missing file. Which I guess is not the end of the world, but it would be nicer to fail explicitly if something is missing. Just a thought.

---

This could happen with each of these files. Should we check all of them? If we take the assumption that the rest of the files will be there because they are extracted on a previous task, we can do the same for the .sig file, since that means that the signing task failed and we won't get here

---

Yeah, ok, maybe it's not needed. So how about setting `failglob` at least? I think that would still be better. That way if one of the patterns doesn't have a match, bash will fail (instead of passing the parameters including wildcards in them).
```sh
$ shopt -s failglob
$ echo *.non-existing
bash: no match: *.non-existing
```

---

Great idea! Added